devices/sonoff.rst

```diff
@@ -309,6 +309,18 @@ Shelly 2
     GPIO5, Relay #2,
     GPIO12, SW Input #1,
     GPIO14, SW Input #2,
+    
+Shelly RGBW2
+--------
+
+.. pintable::
+
+    GPIO12, LED Red,
+    GPIO15, LED Green,
+    GPIO14, LED Blue,
+    GPIO04, LED White,
+    GPIO??, SW Input,
+    GPIO??, Power Meter,
```

Comment:
Do not add information here if we don't know the actual pin.

Also, the power meter chip on the Shelly is not supported - add a line below saying that power metering is not supported at this point.

Reply:
So do I either remove the unknown pins or add information about the unsupported power meter?

Reply:
Both would be best :)  (so remove the unknown pins and add a quick comment below as for the power meter chip not being supported)